packages/ui/nuxt.config.js

```diff
@@ -129,6 +131,65 @@ export default {
     'nuxt-i18n',
     '@nuxtjs/apollo',
     'nuxt-element-ui',
+    '@nuxtjs/feed',
+  ],
+
+  feed: [
+    {
+      path: '/feed.xml', // The route to your feed.
+      async create(feed) {
+        let dataFeeds
+        feed.options = {
+          title: 'Witnet Data Feed Explorer',
+          link: 'https://feeds.witnet.io/feed.xml',
+          description: 'This is the Witnet data feed explorer feed!',
+        }
+        const feedsQuery = `query feeds {
+          feeds (network: "all") {
+            feeds {
+              feedFullName
+              name
+              chain
+              logo
+              networkName
+              heartbeat
+              finality
+              deviation
+              isRouted
+            }
+            total
+          }
+        }`
+        await axios({
+          url: process.env.API_ENDPOINT,
+          method: 'post',
+          data: {
+            query: feedsQuery,
+          },
+        }).then((result) => {
+          dataFeeds = result.data.data.feeds.feeds
+        })
```

Comment:
I don't love the idea of adding a new dependency to make graphql calls "by hand" because we already have those dependencies in our stack. Ideally we should be able to reuse the apollo provider we are using in the whole app. Sadly, I don't see an easy way to use it from the configuration file :disappointed: